app/pb_kits/playbook/packs/kits/pb_project.js

```diff
@@ -0,0 +1 @@
+import '../../pb_project/_project.jsx'
```

Comment:
This file is not used. We need to update the generator. Can you delete it? CC: @thestephenmarshall

Reply:
The file has been removed.
Changes here 2a431c6